mlir/include/mlir/Dialect/MIOpen/Tuning/GridwiseGemmParams.h

```diff
@@ -939,6 +939,7 @@ class PopulateParamsXDL : public PopulateParamsBase {
                    << "\n";
       return failure();
     } else if ((dataType.isF16() || dataType.isBF16()) &&
+               (param.gemmKPack != 1) &&
                ((param.gemmKPack < 4) || (param.gemmKPack > 8))) {
```

Comment:
I think if we want to be explicit, we can explicitly reject 2 and allow everything including 1,4,8:

```cpp
// KPACK of 2 is too narrow for XDLOPS f16 instructions
((param.gemmKPack == 2) && (param.gemmKPack > 8))
```